ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -48,33 +46,26 @@
   >
     <F.Label>{{t 'form.description.label'}}</F.Label>
     <F.HelperText>{{t 'form.description.help'}}</F.HelperText>
-    {{#if @model.errors.description}}
-      <F.Error as |E|>
-        {{#each @model.errors.description as |error|}}
-          <E.Message>{{error.message}}</E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <form.input
-    @name='issuer'
-    @type='text'
+  <Hds::Form::TextInput::Field
     @value={{@model.issuer}}
-    @label={{t 'form.issuer.label'}}
-    @helperText={{t 'form.issuer.help'}}
-    @error={{@model.errors.issuer}}
-    readonly={{false}}
-    as |field|
+    @isInvalid={{@model.errors.issuer}}
+    disabled={{form.disabled}}
+    name='issuer'
```

Comment:
None blocking comment: as in previous implementation, I will suggest keep the attribute `name` the first attribute to facilitate Developer experience (facilitates reading).

Reply:
@calcaide good point

Reply:
@calcaide resolve these in last commits